auto/unix

```diff
@@ -312,6 +312,17 @@ ngx_feature_test="cpu_set_t mask;
 . auto/feature
 
 
+ngx_feature="IP_TRANSPARENT"
+ngx_feature_name="NGX_HAVE_TRANSPARENT"
+ngx_feature_run=no
+ngx_feature_incs="#include <sys/socket.h>
+                  #include <netinet/in.h>"
+ngx_feature_path=
+ngx_feature_libs=
+ngx_feature_test="setsockopt(0, SOL_IP, IP_TRANSPARENT, NULL, 0)"
+. auto/feature
+
+
```

Comment:
This is obviously an old patch ~12yo. Meanwhile about 9 years ago we added transparent proxy support be79f5cb16eeb452e2a9e343a89f89b3b47bc5a2. In that patch there's already a feature check like this, so this one can be removed.

Reply:
Modified accordingly and rebased. Thanks!